modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,21 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Advanced Code Editor 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.3.0 includes the following fix.
+
+=== The `copy code` button wasn't correctly styled within the Advanced Code Editor.
+//#TINY-9698
+
+In previous versions of {productname}, the "Copy code" button had a grey background.
+
+This issue was caused by the previous setting of `borderless: true` which should have been set to `false`. As a consequence, the button wasn't correctly styled. 
```

Comment:
```suggestion
In {productname}, implementation for the new "Copy code" button was styled incorrectly with a grey background.

This issue was a oversight which meant the setting for `borderless: true` should have been set to `false`. As a consequence, the button wasn't correctly styled. 
```

Reply:
@lorenzo-pomili Please review suggested changes, and remove blocker from this if you are happy with the changes.

Reply:
@kemister85 the incorrect style was the one without the grey background